doc/SETUP.md

```diff
@@ -52,14 +52,9 @@ git clone https://github.com/jasongin/nvs "$env:NVS_HOME"
 After installation, running just `nvs` from the current shell or any new Command Prompt or PowerShell then invokes either the `nvs.cmd` or `nvs.ps1` script accordingly.
 
 ### Git Bash on Windows
-NVS can work in Git Bash on Windows (the bash tools installed by [Git](https://git-scm.com/) for Windows), though it requires some manual configuration:
+NVS can work in Git Bash on Windows (the bash tools installed by [Git](https://git-scm.com/) for Windows) automatically.
 
-1. Install NVS using the either the Windows MSI, Command Prompt, or PowerShell instructions above.
-
-2. Ensure there is a `.bash_profile` file in your home (`%HOMEDRIVE%%HOMEPATH%`) directory that calls `.bashrc`. Create the file if it doesn't exist. It should include at least the following line:
```

Comment:
This step might still be worth mentioning, because some users have reported problems from not having `bash_profile` set up to source `.bashrc`, as in https://github.com/jasongin/nvs/issues/211 - I think that's even more likely to be a problem with a git bash environment.